src/components/organisms/ConversationWindow.vue

```diff
@@ -43,23 +43,26 @@ export default {
   },
   setup() {
     const store = useStore();
-    const chatbotObject = computed(
-      () => store.getters["chatMessages/getChatMessageObject"]
-    );
-    const chatbotConvo = computed(
-      () => store.getters["chatMessages/getChatMessagesOrderedByDate"]
+    const selectedInboxItemId = store.getters["inbox/getSelectedInboxItem"].id;
+
+    const chatMessage = computed(() =>
+      store.getters["chatMessages/getChatMessageByIdOrderedByMessagesDate"](
+        selectedInboxItemId
+      )
     );
+
     function sendMessage(msg) {
       const newMessage = {
-        chatId: chatbotObject.value.chatId,
+        chatId: chatMessage.value.chatId,
```

Comment:
This will need to be changed to id instead of chatId

Reply:
Changed (In multiple places)